sdks/java/maven-archetypes/examples-java8/generate-sources.sh

```diff
@@ -22,7 +22,7 @@
 # Usage: Invoke with no arguments from any working directory.
 
 # The directory of this script. Assumes root of the maven-archetypes module.
-HERE="$(dirname $0)"
+HERE="$( cd "$( dirname '$0' )" && pwd )"
```

Comment:
I don't think this will actually work. The `'$0'` will be interpreted literally, so the `dirname` is just `.` and the `cd` doesn't do anything, and `pwd` outputs the current working directory. Either I don't understand this bit of bash, or maven changes the directory, or our tests just don't catch this failure.

I think `"$(dirname \"$0\")"` should do it.